packages/frontend/core/src/modules/at-menu-config/services/index.ts

```diff
@@ -429,40 +429,29 @@ export class AtMenuConfigService extends Service {
 
   // only search docs by title, excluding blocks
   private searchDocs$(query: string) {
-    return this.docsSearch.indexer.blockIndex
+    return this.docsSearch.indexer.docIndex
```

Comment:
will it affect cmdk search?

Reply:
No, this search query is only used by the at menu config